crates/solvers/config/example.balancer.toml

```diff
@@ -1,5 +1,6 @@
 absolute-slippage = "40000000000000000" # Denominated in wei, optional
 relative-slippage = "0.1" # Percentage in the [0, 1] range
+risk-parameters = [1,2,3,4]
```

Comment:
Can we remove those until we actually use them?

Reply:
You mean completely remove all the changes with regards to risk parameters usage in `solvers` crate?

Reply:
Done in https://github.com/cowprotocol/services/pull/1856/commits/d43b05c5934a27349fb134d512634701401fba05
Revert tracked in https://github.com/cowprotocol/services/issues/1899